CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
is this true?

Reply:
It's only a goal. The summary on that page is the primary thing I believe we should strive for.

Reply:
then i think this statement is too strong as is.

Reply:
Agreed. Will reword it.

Reply:
Updated.

Please comment on the latest changes so we avoid confusion.